_assignments/mp1.md

```diff
@@ -4,38 +4,380 @@ title: "MP1: The Room"
+    points: 20
+    description: Built nontrivial room geometry with blender, sketchup, or probuilder
+  
+  -
+    name: 1.2.2
+  -
+    name: Experience 1
+    points: 5
+    description: Likes and dislikes for an Oculus/Steam experience
+  -
+    name: Experience 2
+    points: 5
+    description: Likes and dislikes for a student experience
+  -
+    name: Experience 3
+    points: 5
+    description: Likes and dislikes for any experience
+  -
+    name: Experience 4
+    points: 5
+    description: Likes and dislikes for any experience
+
+
 ---
 
+<!-- this page only, for markdownified TeX -->
+<script src='https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/latest.js?config=TeX-MML-AM_CHTML' async></script>
+
 ## MP1.1:
 Create a new project by clicking through the prompts when you click the NEW button on the top right. Fill in the appropriate details to your heart’s content.
 
+![splash screen](/img/assignments/mp1/image23.png)
+
 Next, read through the following Unity Tutorial, to familiarize yourself with the very basics of the Unity interface:
 
 - [Basic Interface Tutorial](https://docs.unity3d.com/Manual/LearningtheInterface.html)
 - [More In Depth Interface Tutorial](https://docs.unity3d.com/Manual/UsingTheEditor.html)
 
 Feel free to look through some more tutorials, to familiarize yourself with the basics of Unity.
 
+### 1.1.1
+
+![Oh hai Mark!](/img/assignments/mp1/image16.png)
+
+**The Room:** Oh hai Mark!
+
+*You will build a cubic room, whose sides are made out of six planes. 
+Make sure to orient these planes so the visible sides face inwards, and ensure that the player cannot walk through any of them. 
+The room should be 15x15x15 Unity units (aka meters).*
+
+First, create a plane. It can be found in the top bar menus, under `GameObject` → `3D Object`- > `Plane`, as shown below.
+
+![Create plane](/img/assignments/mp1/image3.png)
+
+Now, by default the plane is 10x10 $$ (X \times Z) $$ units. In order to make your room 15 units wide, you have to scale the plane. On the right side (in the default editor layout) you will find the Inspector window. This window provides details about the currently selected object. Select the plane in the Scene view, and the Inspector will fill with information and settings for said plane. Find the “Scale” option, and set it to 1.5 on the $$ X $$ and $$ Z $$ to make your plane 15 units wide and long. 
+
+![Inspector](/img/assignments/mp1/image7.png)  
+Note: The plane has no thickness, so the value in $$ Y $$ can be any positive integer.
+
+By default, your scene has a directional light in it. This is basically the sun- a light source that illuminates your entire scene from a specified angle, from very far away. You’ll notice that your planes do not block this light. That’s because planes only block light (and render) from one side. Bear this in mind when creating objects in Unity in the future! Please delete the directional light (don’t worry, youll add new lights in later)
```

Comment:
missing a period.

Reply:
a period at the very end of this sentence.

Reply:
fixed!